subsys/net/lib/sockets/sockets_select.c

```diff
@@ -106,26 +73,59 @@ int zsock_select(int nfds, zsock_fd_set *readfds, zsock_fd_set *writefds,
 	int i, res, poll_timeout;
 	int num_pfds = 0;
 	int num_selects = 0;
+	int fd_no;
 
-	res = fd_set_to_pollfd(ZSOCK_POLLIN, readfds, pfds, &num_pfds);
-	if (res == -1) {
-		return -1;
-	}
+	fd_no = 0;
+	for (i = 0; i < STRUCT_MEMBER_ARRAY_SIZE(zsock_fd_set, bitset); i++) {
```

Comment:
have an empty line befor for() since you do not use fd_no in its parameters.

Reply:
> have an empty line befor for() since you do not use fd_no in its parameters.

Well, actually fd_no is a secondary loop induction variable, so having straight before the loop makes sense. Here can hove move assignment to var declaration, will do that instead.